lib/config.coffee

```diff
@@ -86,7 +78,6 @@ exports.generate = (options, params = {}) ->
 		vpnPort: 443
 
 	config =
-		applicationName: options.application.app_name
```

Comment:
I wonder if the Supervisor uses that for anything. Might worth checking with Cameron

Reply:
There weren't any references in the latest supervisor code, and I don't think it would have ever been used for anything except maybe analytics (it makes no sense considering you can move apps, and you provision to an app by id not name)

Reply:
I was thinking maybe it's looking for this adding it into its local database. I too can't think of a good use-case for that, but I could be missing something.